video/graphics.h

```diff
@@ -347,9 +346,9 @@ void plotParallelogram() {
 void plotCircle(bool filled = false) {
 	auto size = 2 * sqrt(rp1.X * rp1.X + rp1.Y * rp1.Y);
 	if (filled) {
-		canvas->fillEllipse(p2.X, p2.Y, size, size);
+		canvas->fillEllipse(p2.X, p2.Y, size, rectangularPixels ? size / 2 : size);
```

Comment:
I'd noticed that the random circles demo (`circles_2.bas`) would draw elongated circles when in 640x240 resolution screen modes.  thinking this thru, it's because fab-gl doesn't draw circles, it draws squished circles (ellipses).  this code wasn't caring about pixel shape, and was assuming all pixels are square

our rectangular pixel modes are all 2x1, so this fixes their shape